R/MsExperiment.R

```diff
@@ -89,3 +89,116 @@ setMethod("readMsObject",
               validObject(object)
               object
           })
+
+################################################################################
+##
+## alabaster saveObject/readObject
+##
+################################################################################
+#' @rdname AlabasterParam
+setMethod("saveObject", "MsExperiment", function(x, path, ...) {
+    ## Check requirements before saving
+    if (inherits(x@qdata, "QFeatures"))
```

Comment:
should this  check also be in the PlainTextParam method ? I did not think of that

Reply:
No. For the `MsExperiment` and `PlainTextParam` I would suggest for now to **only** save the sample annotations, MS data (`Spectra`) and the link between samples and spectra. But since there is already an alabaster export of `SummarizedExperiment` I wanted to support that too.